### ISSUE TITLE
Broken docs styles

### ISSUE THREAD

**Issue by marcgarreau** [ethereum/web3.py#1968]
### What was wrong?

Some of the styling in the docs broke recently.

For example, landing page should look like: 

<img width="1101" alt="Screen Shot 2021-04-28 at 12 45 41 PM" src="https://user-images.githubusercontent.com/3621728/116456747-d2432380-a81f-11eb-9712-c8f59bc0e320.png">

but instead looks like:

<img width="1090" alt="Screen Shot 2021-04-28 at 12 45 52 PM" src="https://user-images.githubusercontent.com/3621728/116456778-dcfdb880-a81f-11eb-9648-2aaf4ffba1c1.png">

### How can it be fixed?

Unsure yet, but building the docs locally works fine (`make docs`).

**Comment by tserg** [comment]
I tried running `make doctest` locally and got the following errors:

> WARNING: Failed to get a method signature for ethpm.Package.build_dependencies: <ethpm._utils.cache.cached_property object at 0x7f50b9674ac0> is not a callable object
WARNING: Failed to get a method signature for ethpm.Package.deployments: <ethpm._utils.cache.cached_property object at 0x7f50b9674b20> is not a callable object

Making the following changes to `ethpm.rst` resolves the errors:

Original
```
.. autoclass:: ethpm.Package
   :members: name, version, manifest_version, uri, __repr__, contract_types, build_dependencies, deployments
```
Amended

```
.. autoclass:: ethpm.Package
   :members: name, version, manifest_version, uri, __repr__, contract_types

   .. automethod:: build_dependencies(self)
   .. automethod:: deployments(self)
```

I am not sure if this is causing the errors though.

**Comment by kclowes** [comment]
This has been fixed in #1977!
